articles/quickstart/native/android/03-session-handling.md

```diff
@@ -18,24 +18,25 @@ This tutorial will show you how to login and maintain an active session with Aut
   ]
 }) %>__
 
-For this, you will need to handle the user's `Credentials`. Let's take a look at this class, which is composed of five objects:
+You need the `Credentials` class to handle users' credentials. The class is composed of seven objects:
 
-* `accessToken`: Access Token used by the Auth0 API.
-* `idToken`: Identity Token that proves the identity of the user.
-* `refreshToken`: Refresh Token that can be used to request new tokens without signing in again.
-* `tokenType`: The type of the tokens issued by the server.
-* `expiresIn`: The amount in seconds in which the tokens will be deemed invalid.
-* `expiresAt`: The Date in which the tokens will be deemed invalid.
-* `scope`: The granted scope, if different from the requested one.
+* `accessToken`: Access token used by the Auth0 API. To learn more, see the [access token documentation](/tokens/access-token).
+* `idToken`: Identity token that proves the identity of the user. To learn more, see the [ID token documentation](/tokens/id-token).
+* `refreshToken`: Refresh token that can be used to request new tokens without signing in again. To learn more, see the [refresh token documentation](/tokens/refresh-token/current).
+* `tokenType`: The type of tokens issued by the server.
+* `expiresIn`: The amount of seconds before the tokens expire.
+* `expiresAt`: The date when the tokens expire.
+* `scope`: The scope that was granted to a user. This information is shown only if the granted token is different than the requested one.
 
-The Tokens are the objects used to prove your identity against the Auth0 APIs. Read more about them [here](https://auth0.com/docs/tokens).
+The tokens are the objects used to prove your identity against the Auth0 APIs. Read more about them in the [tokens documentation](https://auth0.com/docs/tokens).
 
+## Before You Start
```

Comment:
There are two general types of headers in the quickstarts: 
* Steps in the process
* Descriptions 

To introduce some separation between the two types, how about: 
* Using imperative for the steps: "Do X", "Configure Y"
* Using the "About X" form for descriptions? 

That way, the readers will have some idea of what to expect from the section following the header. 

@aaguiarz